src/index.ts

```diff
@@ -220,18 +220,15 @@ const getCompareValue = (
 };
 
 /**
- * @returns `true` if the comparision matches, `false` if there is a mismatch,
+ * @returns `true` if the comparison matches, `false` if there is a mismatch,
  *           and `undefined` if the target value is not known to compute the
  *           result.
  */
 const compare = (
-  condition: AbacComparison | undefined,
```

Comment:
@swain I'm pretty sure there are some projects that are sending in `undefined` here, but I can't for sure remember.  It doesn't make sense to me to be using this with undefined, so maybe it will uncover some bugs.

There was no coverage of line 233 in the tests, and I'm not sure it is possible to reach the line.